TTS/trainer.py

```diff
@@ -876,7 +908,7 @@ def eval_step(self, batch: Dict, step: int) -> Tuple[Dict, Dict]:
         with torch.no_grad():
             outputs = []
             loss_dict = {}
-            if not isinstance(self.optimizer, list):
+            if not isinstance(self.optimizer, list) or self.model.capacitron_vae:
```

Comment:
Since in the capacitron case, `self.optimizer` **is** an instance of a list, this condition hacks this again a little bit.